src/scaffoldmaker/utils/tubenetworkmesh.py

```diff
@@ -1050,6 +1261,137 @@ def setRimElementId(self, e1, e2, e3, elementIdentifier):
             self._rimElementIds[e2] = [[None] * self._elementsCountAround for _ in range(elementsCountRim)]
         self._rimElementIds[e2][e3][e1] = elementIdentifier
 
+    def getBoxElementId(self, e1, e2, e3):
+        """
+        Get a box element ID.
+        :param e1: Element index across core box major / d2 direction.
+        :param e2: Element index along segment.
+        :param e3: Element index across core box minor / d3 direction.
+        :return: Element identifier.
+        """
+        return self._boxElementIds[e2][e3][e1]
+
+    def setBoxElementId(self, e1, e2, e3, elementIdentifier):
+        """
+        Set a box element ID. Only called by adjacent junctions.
+        :param e1: Element index across core box major / d2 direction.
+        :param e2: Element index along segment.
+        :param e3: Element index across core box minor / d3 direction.
+        :param elementIdentifier: Element identifier.
+        """
+        self._elementsCountCoreBoxMajor
```

Comment:
Lines 1282 and 1283 are not doing anything so I think we can remove them.

Reply:
Removed.